lib/dpul_collections_web/live/home_live.ex

```diff
@@ -21,15 +24,18 @@ defmodule DpulCollectionsWeb.HomeLive do
           <div class="grid grid-cols-4">
             <input class="col-span-4 md:col-span-3" type="text" name="q" value={@q} />
             <button class="col-span-4 md:col-span-1 btn-primary" type="submit">
-              Search
+              <%= gettext("Search") %>
             </button>
           </div>
         </form>
       </div>
       <div id="welcome" class="grid place-self-center gap-10 max-w-prose">
-        <h3 class="text-5xl text-center">Explore Our Digital Collections</h3>
+        <h3 class="text-5xl text-center"><%= gettext("Explore Our Digital Collections") %></h3>
         <p class="text-xl text-center">
-          We invite you to be inspired by our globally diverse collections of <%= @item_count %> Ephemera items. We can't wait to see how you use these materials to support your unique research.
+          <%= gettext("We invite you to be inspired by our globally diverse collections of") %> <%= @item_count %>
+          <%= gettext(
```

Comment:
Does the break between tags render as a newline?

Reply:
It does. The text is very long. I usually have soft wrapping on in my editor, but I try to avoid lines that are that long. I am not tied to having a line break here.

Reply:
There's a max width style on it so I don't think it will ever get too long on the actual screen.